src/components/ForgotPasswordEmail.js

```diff
@@ -0,0 +1,21 @@
+//TODO: create a email input
+import React from "react"
+import { Form, FormGroup, Label, Input } from "reactstrap"
+
+export default class ForgotPasswordEmail extends React.Component {
+  render() {
+    return (
+      <Form
```

Comment:
problem with the Form tag here because the button should be inside this tag, move Form to a parent component